README.md

```diff
@@ -213,7 +213,48 @@ executable `taskDPL`.
 
 ## Data Sampling
 
-TODO -> piotr
+Data Sampling provides possibility to sample data in DPL workflows, basing on certain conditions ( 5% randomly, when payload is greater than 4234 bytes, etc.). The job of passing the right data is done by a data processor called `Dispatcher`. A desired data stream is specified in form of Data Sampling Policies, configured by JSON structures. Please refer to the main [Data Sampling readme](https://github.com/AliceO2Group/AliceO2/blob/dev/Framework/Core/README.md#data-sampling) for more detailed information.
+
+Data Sampling is used by Quality Control to feed tasks with data. Below, an exemplary usage in configuration file is presented:
+```json
+{
+  "qc": {
+    ...
+    "tasks_config": {
+      "QcTask": {
+        "taskDefinition": "QcTaskDefinition"
+      },
+      "QcTaskDefinition": {
+        ...
+        "dataSamplingPolicy": "its-raw"
+      }
+    }
+  },
+  "dataSamplingPolicies": [
+    {
+      "id": "its-raw",
+      "active": "true",
+      "machines": [],
+      "dataHeaders": [
+        {
+          "binding": "raw",
+          "dataOrigin": "ITS",
+          "dataDescription": "RAWDATA"
+        }
+      ],
+      "subSpec": "0",
+      "samplingConditions": [
+        {
+          "condition": "random",
+          "fraction": "0.1",
+          "seed": "1234"
+        }
+      ],
+      "blocking": "false"
+    }
+  ]
+}
+```
 
```

Comment:
I am wondering if we should describe the config in details. At least the different samplingConditions ? or point to another document that describes the config ?

Reply:
It is described in the main Data Sampling readme, there is an example config with each parameter commented and a list of sampling conditions. I wanted to avoid duplicating the information, since it could diverge at some point.